docs/index.md

```diff
@@ -1087,16 +1087,36 @@ Files specified using `--file` _are not affected_ by this option.
 
 Can be specified multiple times.
 
-### `--extension <ext>, --watch-extensions <ext>`
-
-> _Updated in v6.0.0. Previously `--watch-extensions`, but now expanded to affect general test file loading behavior. `--watch-extensions` is now an alias_
+### `--extension <ext>`
 
 Files having this extension will be considered test files. Defaults to `js`.
 
-Affects `--watch` behavior.
-
 Specifying `--extension` will _remove_ `.js` as a test file extension; use `--extension js` to re-add it. For example, to load `.mjs` and `.js` test files, you must supply `--extension mjs --extension js`.
 
+The option can be given multiple times. The option accepts a comma-delimited list: `--extension a,b` is equivalent to `--extension a --extension b`
+
+### `--watch-files <file|directory|glob>`
```

Comment:
Please move `--watch-files <file|directory|glob>` and `--watch-ignore <file|directory|glob>` documentation below `--watch, -w`. Just the same order as in the help output.

Reply:
https://github.com/mochajs/mocha/pull/3980/commits/7cccd07725a9a72b1708b4983b30226cf3997c96

Reply:
There is one empty line too much before "--fgrep" (L1159), which results in a linting error.